dataikuapi/dss/projectfolder.py

```diff
@@ -0,0 +1,236 @@
+from .project import DSSProject
+
+class DSSProjectFolder(object):
+    """
+    A handle to interact with a project folder on the DSS instance.
+
+    Do not create this class directly, instead use :meth:`dataikuapi.DSSClient.get_project_folder` or :meth:`dataikuapi.DSSClient.get_root_project_folder`
+    """
+    def __init__(self, client, project_folder_id):
+        self.client = client
+        self.project_folder_id = project_folder_id
+
+    ########################################################
+    # Project folder basics
+    ########################################################
+    def get_name(self):
+        """
+        Get this project folder's name or None if it is the root project folder
+
+        :returns str: the name of this project folders or None for the root project folder
+        """
+        return self.client._perform_json("GET", "/project-folders/%s" % self.project_folder_id).get("name", None)
+
+    def get_path(self):
+        """
+        Get this project fodler's path based on the root project folder
+
+        :returns str: the path of this project folder
+        """
+        definition = self.client._perform_json("GET", "/project-folders/%s" % self.project_folder_id)
+        parent_id = definition.get("parent", None)
+        if parent_id is not None:
+            parent = DSSProjectFolder(self.client, parent_id)
+            path = parent.get_path()
+            return ("" if path == "/" else path) + "/" + definition.get("name", "")
+        else:
+            return "/"
+
+    def get_parent(self):
+        """
+        Get this project folder's parent or None if it is the root project folder
+
+        :returns: A :class:`dataikuapi.dss.projectfolders.DSSProjectFolder` to interact with its parent or None for the root project folder
+        """
+        parent = self.client._perform_json("GET", "/project-folders/%s" % self.project_folder_id).get("parent", None)
+        if parent is None:
+            return None
+        else:
+            return DSSProjectFolder(self.client, parent)
+
+    def get_children(self):
```

Comment:
For consistency, this method shoudl be called `list_something` rather than `get_something`.

To avoid doubts about whether this includes projects or not, I'd call it `list_child_folders`